community_images/kafka/ironbank/image.yml

```diff
@@ -0,0 +1,44 @@
+name: kafka-ib
+official_name: Apacha Kafka Ironbank
+official_website: https://kafka.apache.org/
+source_image_provider: Platform One
+source_image_repo: registry1.dso.mil/ironbank/bitnami/kafka
+source_image_repo_link: https://registry1.dso.mil/harbor/projects/3/repositories/opensource%2Fbitnami%2Fkafka
```

Comment:
Please remove `opensource`. Bitnami does not come under opensource project/directory on repo1 or registry1.